website/components/mdx/_download-linux.mdx

```diff
@@ -5,7 +5,7 @@ import { Tabs, Tab } from '../Tabs'
 <Tab>
 
 ```bash copy
-curl -L https://github.com/cloudquery/cloudquery/releases/download/VERSION_CLI/cloudquery_linux_amd64 -o cloudquery
```

Comment:
Not having `cli-` prefix here is a bug also on https://www.cloudquery.io/

Reply:
Extracted this to a separate PR https://github.com/cloudquery/cloudquery/pull/5484